pages/portfolio.html

```diff
@@ -5,8 +5,47 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="/styles/porfolio_style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header>
+    <h1>Portfolio</h1>
+    <nav>
+      <ul>
+        <li><a href="/pages/index.html">Home</a></li>
+        <li><a href="/pages/about.html">About</a></li>
+        <li><a href="/pages/blog.html">Blog</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="container">
+    <section class="photography">
+      <h2>Photography</h2>
+      <div>
+        <div>
+          <img src ="/images/Earlhamite2.jpg" alt="scan of silver gelatin print of farm building at Earlham College "/>
+        </div>
```

Comment:
Like in the index, I'd try to remove this intermediate div layer. If you want to group the images together under a div (maybe labelled with a `gallery` class), that can help provide an anchoring point for rules. Or we could also think of this as being a list of images and link about using ul li tags which would be more meaningful.